```
                                                                     ^
/home/runner/work/fineract/fineract/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/modules/ClientRequestBuilders.java:38: error: cannot find symbol
                .submittedOnDate(submittedOnDate)//
                ^
  symbol:   method submittedOnDate(String)
  location: class PostClientsRequest
/home/runner/work/fineract/fineract/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/modules/ClientRequestBuilders.java:77: error: cannot find symbol
                .withdrawalReasonId(withdrawalReasonId)//
                ^
  symbol:   method withdrawalReasonId(Long)
  location: class PostClientsClientIdRequest
/home/runner/work/fineract/fineract/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/modules/ClientRequestBuilders.java:85: error: cannot find symbol
                .reopenedDate(reopenedDate);
                ^
  symbol:   method reopenedDate(String)
  location: class PostClientsClientIdRequest
/home/runner/work/fineract/fineract/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/modules/ClientRequestBuilders.java:92: error: cannot find symbol
                .reopenedDate(reopenedDate);
                ^
  symbol:   method reopenedDate(String)
  location: class PostClientsClientIdRequest
/home/runner/work/fineract/fineract/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/modules/ClientRequestBuilders.java:97: error: cannot find symbol
                .firstname(firstname)//
                ^
```

Please run:
`./gradlew spotlessApply spotbugsMain spotbugsTest checkstyleMain checkstyleTest`
`./gradlew --no-daemon build -x test -x cucumber -x doc`

--- a/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/helpers/FeignClientHelper.java
+++ b/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/helpers/FeignClientHelper.java
@@ -18,17 +18,37 @@
  */
 package org.apache.fineract.integrationtests.client.feign.helpers;
 
+import static org.apache.fineract.client.feign.util.FeignCalls.fail;
 import static org.apache.fineract.client.feign.util.FeignCalls.ok;
 
 import java.util.Collections;
 import org.apache.fineract.client.feign.FineractFeignClient;
+import org.apache.fineract.client.feign.util.CallFailedRuntimeException;
+import org.apache.fineract.client.models.ClientTextSearch;
+import org.apache.fineract.client.models.DeleteClientsClientIdResponse;
+import org.apache.fineract.client.models.GetClientsClientIdAccountsResponse;
 import org.apache.fineract.client.models.GetClientsClientIdResponse;
+import org.apache.fineract.client.models.PageClientSearchData;
+import org.apache.fineract.client.models.PagedRequestClientTextSearch;
+import org.apache.fineract.client.models.PostClientsClientIdRequest;
+import org.apache.fineract.client.models.PostClientsClientIdResponse;
 import org.apache.fineract.client.models.PostClientsRequest;
 import org.apache.fineract.client.models.PostClientsResponse;
+import org.apache.fineract.client.models.PutClientsClientIdRequest;
+import org.apache.fineract.client.models.PutClientsClientIdResponse;
+import org.apache.fineract.integrationtests.client.feign.modules.ClientRequestBuilders;
 import org.apache.fineract.integrationtests.client.feign.modules.LoanTestData;
 import org.apache.fineract.integrationtests.common.Utils;
 
 public class FeignClientHelper {
+
+    private static final String ACTIVATE_COMMAND = "activate";
+    private static final String CLOSE_COMMAND = "close";
+    private static final String REJECT_COMMAND = "reject";
+    private static final String REACTIVATE_COMMAND = "reactivate";
+    private static final String WITHDRAW_COMMAND = "withdraw";
+    private static final String UNDO_REJECTION_COMMAND = "undoRejection";
+    private static final String UNDO_WITHDRAWAL_COMMAND = "undoWithdrawal";
 
     private final FineractFeignClient fineractClient;
 
@@ -54,15 +74,78 @@ public class FeignClientHelper {
                 .dateFormat(LoanTestData.DATETIME_PATTERN)//
                 .locale(LoanTestData.LOCALE);
 
-        return createClient(request);
+        return createClient(request).getClientId();
     }
 
-    public Long createClient(PostClientsRequest request) {
-        PostClientsResponse response = ok(() -> fineractClient.clients().createClient(request));
-        return response.getClientId();
+    public PostClientsResponse createClient(PostClientsRequest request) {
+        return ok(() -> fineractClient.clients().createClient(request));
+    }
+
+    public PostClientsResponse createClientPending() {
+        return createClientPending(Utils.dateFormatter.format(Utils.getLocalDateOfTenant()));
+    }
+
+    public PostClientsResponse createClientPending(String submittedOnDate) {
+        return createClientPending(ClientRequestBuilders.createPendingClient(submittedOnDate));
+    }
+
+    public PostClientsResponse createClientPending(PostClientsRequest request) {
+        return ok(() -> fineractClient.clients().createClient(request));
     }
 
     public GetClientsClientIdResponse getClient(Long clientId) {
         return ok(() -> fineractClient.clients().retrieveOneClient(clientId, Collections.emptyMap()));
+    }
+
+    public CallFailedRuntimeException getClientExpectingError(Long clientId) {
+        return fail(() -> fineractClient.clients().retrieveOneClient(clientId, Collections.emptyMap()));
+    }
+
+    public GetClientsClientIdAccountsResponse getClientAccounts(Long clientId) {
+        return ok(() -> fineractClient.clients().retrieveAllClientAccounts(clientId));
+    }
+
+    public PageClientSearchData searchClients(String text) {
+        ClientTextSearch clientTextSearch = new ClientTextSearch();
+        clientTextSearch.setText(text);
+        PagedRequestClientTextSearch request = new PagedRequestClientTextSearch();
+        request.setRequest(clientTextSearch);
+        return ok(() -> fineractClient.clientSearchV2().searchClientsByText(request));
+    }
+
+    public PostClientsClientIdResponse activateClient(Long clientId, PostClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().handleCommandClient(clientId, request, ACTIVATE_COMMAND));
+    }
+
+    public PostClientsClientIdResponse closeClient(Long clientId, PostClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().handleCommandClient(clientId, request, CLOSE_COMMAND));
+    }
+
+    public PostClientsClientIdResponse rejectClient(Long clientId, PostClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().handleCommandClient(clientId, request, REJECT_COMMAND));
+    }
+
+    public PostClientsClientIdResponse reactivateClient(Long clientId, PostClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().handleCommandClient(clientId, request, REACTIVATE_COMMAND));
+    }
+
+    public PostClientsClientIdResponse withdrawClient(Long clientId, PostClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().handleCommandClient(clientId, request, WITHDRAW_COMMAND));
+    }
+
+    public PostClientsClientIdResponse undoRejectClient(Long clientId, PostClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().handleCommandClient(clientId, request, UNDO_REJECTION_COMMAND));
+    }
+
+    public PostClientsClientIdResponse undoWithdrawnClient(Long clientId, PostClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().handleCommandClient(clientId, request, UNDO_WITHDRAWAL_COMMAND));
+    }
+
+    public PutClientsClientIdResponse updateClient(Long clientId, PutClientsClientIdRequest request) {
+        return ok(() -> fineractClient.clients().updateClient(clientId, request));
+    }
+
+    public DeleteClientsClientIdResponse deleteClient(Long clientId) {
+        return ok(() -> fineractClient.clients().deleteClient(clientId));
     }
 }

--- a/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/modules/ClientRequestBuilders.java
+++ b/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/modules/ClientRequestBuilders.java
@@ -1,0 +1,100 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.integrationtests.client.feign.modules;
+
+import org.apache.fineract.client.models.PostClientsClientIdRequest;
+import org.apache.fineract.client.models.PostClientsRequest;
+import org.apache.fineract.client.models.PutClientsClientIdRequest;
+import org.apache.fineract.integrationtests.common.Utils;
+
+public final class ClientRequestBuilders {
+
+    private ClientRequestBuilders() {}
+
+    public static PostClientsRequest createPendingClient(String submittedOnDate) {
+        return new PostClientsRequest()//
+                .officeId(1L)//
+                .legalFormId(1L)//
+                .firstname(Utils.randomFirstNameGenerator())//
+                .lastname(Utils.randomLastNameGenerator())//
+                .externalId(Utils.randomStringGenerator("EXT_", 7))//
+                .active(false)//
+                .submittedOnDate(submittedOnDate)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .locale(LoanTestData.LOCALE);
+    }
+
+    public static PostClientsClientIdRequest activateClient(String activationDate) {
+        return new PostClientsClientIdRequest()//
+                .locale(LoanTestData.LOCALE)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .activationDate(activationDate);
+    }
+
+    public static PostClientsClientIdRequest closeClient(Long closureReasonId, String closureDate) {
+        return new PostClientsClientIdRequest()//
+                .locale(LoanTestData.LOCALE)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .closureReasonId(closureReasonId)//
+                .closureDate(closureDate);
+    }
+
+    public static PostClientsClientIdRequest rejectClient(Long rejectionReasonId, String rejectionDate) {
+        return new PostClientsClientIdRequest()//
+                .locale(LoanTestData.LOCALE)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .rejectionReasonId(rejectionReasonId)//
+                .rejectionDate(rejectionDate);
+    }
+
+    public static PostClientsClientIdRequest reactivateClient(String reactivationDate) {
+        return new PostClientsClientIdRequest()//
+                .locale(LoanTestData.LOCALE)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .reactivationDate(reactivationDate);
+    }
+
+    public static PostClientsClientIdRequest withdrawClient(Long withdrawalReasonId, String withdrawalDate) {
+        return new PostClientsClientIdRequest()//
+                .locale(LoanTestData.LOCALE)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .withdrawalReasonId(withdrawalReasonId)//
+                .withdrawalDate(withdrawalDate);
+    }
+
+    public static PostClientsClientIdRequest undoRejectClient(String reopenedDate) {
+        return new PostClientsClientIdRequest()//
+                .locale(LoanTestData.LOCALE)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .reopenedDate(reopenedDate);
+    }
+
+    public static PostClientsClientIdRequest undoWithdrawnClient(String reopenedDate) {
+        return new PostClientsClientIdRequest()//
+                .locale(LoanTestData.LOCALE)//
+                .dateFormat(LoanTestData.DATETIME_PATTERN)//
+                .reopenedDate(reopenedDate);
+    }
+
+    public static PutClientsClientIdRequest updateClientName(String firstname, String lastname) {
+        return new PutClientsClientIdRequest()//
+                .firstname(firstname)//
+                .lastname(lastname);
+    }
+}

--- a/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/tests/FeignClientLifecycleTest.java
+++ b/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/tests/FeignClientLifecycleTest.java
@@ -1,0 +1,245 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.integrationtests.client.feign.tests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.fineract.client.feign.FineractFeignClient;
+import org.apache.fineract.client.feign.util.CallFailedRuntimeException;
+import org.apache.fineract.client.models.GetClientsClientIdAccountsResponse;
+import org.apache.fineract.client.models.GetClientsClientIdResponse;
+import org.apache.fineract.client.models.GetCodesResponse;
+import org.apache.fineract.client.models.PageClientSearchData;
+import org.apache.fineract.client.models.PostClientsClientIdResponse;
+import org.apache.fineract.client.models.PostClientsResponse;
+import org.apache.fineract.client.models.PostCodeValueDataResponse;
+import org.apache.fineract.client.models.PostCodeValuesDataRequest;
+import org.apache.fineract.client.models.PutClientsClientIdResponse;
+import org.apache.fineract.integrationtests.client.FeignIntegrationTest;
+import org.apache.fineract.integrationtests.client.feign.helpers.FeignClientHelper;
+import org.apache.fineract.integrationtests.client.feign.modules.ClientRequestBuilders;
+import org.apache.fineract.integrationtests.common.FineractFeignClientHelper;
+import org.apache.fineract.integrationtests.common.Utils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class FeignClientLifecycleTest extends FeignIntegrationTest {
+
+    private static final String STATUS_PENDING = "clientStatusType.pending";
+    private static final String STATUS_ACTIVE = "clientStatusType.active";
+    private static final String STATUS_CLOSED = "clientStatusType.closed";
+    private static final String STATUS_REJECTED = "clientStatusType.rejected";
+    private static final String STATUS_WITHDRAWN = "clientStatusType.withdraw";
+
+    private static FeignClientHelper clientHelper;
+    private static FineractFeignClient fineractClient;
+
+    @BeforeAll
+    public static void setup() {
+        fineractClient = FineractFeignClientHelper.getFineractFeignClient();
+        clientHelper = new FeignClientHelper(fineractClient);
+    }
+
+    private Long resolveOrCreateCodeValue(String codeName) {
+        GetCodesResponse code = ok(() -> fineractClient.codes().retrieveCodeByName(codeName));
+        assertNotNull(code.getId());
+
+        PostCodeValuesDataRequest createRequest = new PostCodeValuesDataRequest()//
+                .name(Utils.randomStringGenerator(codeName + "_", 5))//
+                .isActive(true);
+        PostCodeValueDataResponse created = ok(() -> fineractClient.codeValues().createCodeValue(code.getId(), createRequest));
+        return created.getSubResourceId();
+    }
+
+    private void assertClientStatus(Long clientId, String expectedStatusCode) {
+        GetClientsClientIdResponse details = clientHelper.getClient(clientId);
+        assertNotNull(details.getStatus());
+        assertEquals(expectedStatusCode, details.getStatus().getCode());
+    }
+
+    @Test
+    @Order(1)
+    void testCreatePendingAndActivateClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        PostClientsResponse pending = clientHelper.createClientPending(today);
+        assertNotNull(pending.getClientId());
+        assertClientStatus(pending.getClientId(), STATUS_PENDING);
+
+        PostClientsClientIdResponse activateResp = clientHelper.activateClient(pending.getClientId(),
+                ClientRequestBuilders.activateClient(today));
+        assertNotNull(activateResp.getClientId());
+        assertClientStatus(pending.getClientId(), STATUS_ACTIVE);
+    }
+
+    @Test
+    @Order(2)
+    void testCreatePendingAndRejectClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        PostClientsResponse pending = clientHelper.createClientPending(today);
+        assertNotNull(pending.getClientId());
+
+        Long rejectionReasonId = resolveOrCreateCodeValue("ClientRejectReason");
+
+        PostClientsClientIdResponse rejectResp = clientHelper.rejectClient(pending.getClientId(),
+                ClientRequestBuilders.rejectClient(rejectionReasonId, today));
+        assertNotNull(rejectResp.getClientId());
+        assertClientStatus(pending.getClientId(), STATUS_REJECTED);
+    }
+
+    @Test
+    @Order(3)
+    void testRejectAndUndoRejectClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        PostClientsResponse pending = clientHelper.createClientPending(today);
+        assertNotNull(pending.getClientId());
+
+        Long rejectionReasonId = resolveOrCreateCodeValue("ClientRejectReason");
+        clientHelper.rejectClient(pending.getClientId(), ClientRequestBuilders.rejectClient(rejectionReasonId, today));
+        assertClientStatus(pending.getClientId(), STATUS_REJECTED);
+
+        PostClientsClientIdResponse undoResp = clientHelper.undoRejectClient(pending.getClientId(),
+                ClientRequestBuilders.undoRejectClient(today));
+        assertNotNull(undoResp.getClientId());
+        assertClientStatus(pending.getClientId(), STATUS_PENDING);
+    }
+
+    @Test
+    @Order(4)
+    void testWithdrawClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        PostClientsResponse pending = clientHelper.createClientPending(today);
+        assertNotNull(pending.getClientId());
+
+        Long withdrawalReasonId = resolveOrCreateCodeValue("ClientWithdrawReason");
+
+        PostClientsClientIdResponse withdrawResp = clientHelper.withdrawClient(pending.getClientId(),
+                ClientRequestBuilders.withdrawClient(withdrawalReasonId, today));
+        assertNotNull(withdrawResp.getClientId());
+        assertClientStatus(pending.getClientId(), STATUS_WITHDRAWN);
+    }
+
+    @Test
+    @Order(5)
+    void testWithdrawAndUndoWithdrawClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        PostClientsResponse pending = clientHelper.createClientPending(today);
+        assertNotNull(pending.getClientId());
+
+        Long withdrawalReasonId = resolveOrCreateCodeValue("ClientWithdrawReason");
+        clientHelper.withdrawClient(pending.getClientId(), ClientRequestBuilders.withdrawClient(withdrawalReasonId, today));
+        assertClientStatus(pending.getClientId(), STATUS_WITHDRAWN);
+
+        PostClientsClientIdResponse undoResp = clientHelper.undoWithdrawnClient(pending.getClientId(),
+                ClientRequestBuilders.undoWithdrawnClient(today));
+        assertNotNull(undoResp.getClientId());
+        assertClientStatus(pending.getClientId(), STATUS_PENDING);
+    }
+
+    @Test
+    @Order(6)
+    void testCloseAndReactivateClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        Long clientId = clientHelper.createClient(today);
+
+        Long closureReasonId = resolveOrCreateCodeValue("ClientClosureReason");
+
+        PostClientsClientIdResponse closeResp = clientHelper.closeClient(clientId,
+                ClientRequestBuilders.closeClient(closureReasonId, today));
+        assertNotNull(closeResp.getClientId());
+        assertClientStatus(clientId, STATUS_CLOSED);
+
+        PostClientsClientIdResponse reactivateResp = clientHelper.reactivateClient(clientId, ClientRequestBuilders.reactivateClient(today));
+        assertNotNull(reactivateResp.getClientId());
+        assertClientStatus(clientId, STATUS_PENDING);
+    }
+
+    @Test
+    @Order(7)
+    void testUpdateClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        Long clientId = clientHelper.createClient(today);
+
+        String newFirstName = Utils.randomFirstNameGenerator();
+        String newLastName = Utils.randomLastNameGenerator();
+        PutClientsClientIdResponse updateResp = clientHelper.updateClient(clientId,
+                ClientRequestBuilders.updateClientName(newFirstName, newLastName));
+        assertNotNull(updateResp.getClientId());
+
+        GetClientsClientIdResponse details = clientHelper.getClient(clientId);
+        assertEquals(newFirstName, details.getFirstname());
+        assertEquals(newLastName, details.getLastname());
+    }
+
+    @Test
+    @Order(8)
+    void testDeletePendingClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        PostClientsResponse pending = clientHelper.createClientPending(today);
+        assertNotNull(pending.getClientId());
+
+        clientHelper.deleteClient(pending.getClientId());
+
+        CallFailedRuntimeException exception = clientHelper.getClientExpectingError(pending.getClientId());
+        assertEquals(404, exception.getStatus());
+    }
+
+    @Test
+    @Order(9)
+    void testSearchClient() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        String uniqueLastName = Utils.randomStringGenerator("SearchTest_", 8);
+        Long clientId = clientHelper.createClient(today);
+
+        GetClientsClientIdResponse client = clientHelper.getClient(clientId);
+        String searchName = client.getLastname();
+
+        PageClientSearchData results = clientHelper.searchClients(searchName);
+        assertNotNull(results);
+        assertNotNull(results.getContent());
+        assertFalse(results.getContent().isEmpty());
+        assertTrue(results.getContent().stream().anyMatch(r -> r.getId().equals(clientId)));
+    }
+
+    @Test
+    @Order(10)
+    void testGetClientAccounts() {
+        String today = Utils.dateFormatter.format(Utils.getLocalDateOfTenant());
+
+        Long clientId = clientHelper.createClient(today);
+
+        GetClientsClientIdAccountsResponse accounts = clientHelper.getClientAccounts(clientId);
+        assertNotNull(accounts);
+    }
+}

--- a/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/tests/FeignTrialBalanceSummaryReportTest.java
+++ b/integration-tests/src/test/java/org/apache/fineract/integrationtests/client/feign/tests/FeignTrialBalanceSummaryReportTest.java
@@ -351,7 +351,7 @@ public class FeignTrialBalanceSummaryReportTest extends FeignIntegrationTest {
                 .active(true)//
                 .activationDate(activationDate)//
                 .dateFormat(LoanTestData.DATETIME_PATTERN)//
-                .locale(LoanTestData.LOCALE));
+                .locale(LoanTestData.LOCALE)).getClientId();
     }
 
     private Long createAndDisburseLoan(Long clientId, String submitDate, String disburseDate, Long chargeId, String originatorExternalId) {

